snap/snapcraft.yaml

```diff
@@ -6,6 +6,7 @@ license: GPL-3.0
 grade: stable
 confinement: strict
 base: core18
+icon: snap/gui/icon.png
```

Comment:
Does this need to be in the snap directory? Would it make more sense to just use the `res/app/512.png` file? If it needs to be here, why not just use a symlink?

Reply:
The location and icon there is standard location and is recognized by snapcraft everywhere.

Reply:
Wouldn't a symlink work as well? And why use the png icon instead of the svg icon?

Reply:
Symlink will not work, Every desktop app snap uses a PNG 512 icon in the gui folder that's all i know. This make the snap play nice with their build system to listing system.

Reply:
Reverted back the .desktop file.